released/packages/coq-ext-lib/coq-ext-lib.0.11.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/coq-community/coq-ext-lib"
+dev-repo: "git+https://github.com/coq-community/coq-ext-lib.git"
+bug-reports: "https://github.com/coq-community/coq-ext-lib/issues"
+authors: ["Gregory Malecha"]
+license: "BSD"
```

Comment:
Can we get a clarification exactly what the license is for this project, as an SPDX identifier? Based on the repo, it seems to be [BSD-2-Clause-FreeBSD](https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html)?

Reply:
Should I update this field in all previous versions as well?

Reply:
If it's not too much work, sure, but if you want this version merged quickly you should probably do it as a separate PR (since CI will take time).